README: Add link to API reference docs. (#58)
y-scope/clp-ffi-py: README.md

Tested on Python 3.7, 3.8, 3.11 and 3.12, and it should work on any Python
version >= 3.7.

## API Reference

> The API reference of this library can be found in [YScope Docs][19].

Reviewer: ```suggestion
The API reference for this library can be found on our [docs hub][19].
```
